mithril-common/src/chain_observer/model.rs

```diff
@@ -7,6 +8,43 @@ use thiserror::Error;
 
 use crate::{StdError, StdResult};
 
+cfg_fs! {
+    use serde::Deserialize;
+    use pallas_primitives::{alonzo::PlutusData, ToCanonicalJson};
+    /// [Datum] represents an inline datum from UTxO.
+    #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, PartialOrd, Ord, Clone)]
+    #[serde(rename_all = "lowercase")]
+    pub struct Datum(pub PlutusData);
+
+    impl ToCanonicalJson for Datum {
+        fn to_json(&self) -> serde_json::Value {
+            self.0.to_json()
+        }
+    }
+
+    impl<'a, C> minicbor::Decode<'a, C> for Datum {
```

Comment:
Why not use imports for `minicbor`?
IMO it's a bit cumbersome to repeat it in these functions.